src/frequenz/sdk/timeseries/logical_meter/_formula_generators/_ev_charger_power_formula.py

```diff
@@ -0,0 +1,56 @@
+# License: MIT
+# Copyright © 2022 Frequenz Energy-as-a-Service GmbH
+
+"""Formula generator from component graph for Grid Power."""
+
+import logging
+
+from .....sdk import microgrid
+from ....microgrid.component import ComponentCategory, ComponentMetricId
+from .._formula_engine import FormulaEngine
+from ._formula_generator import NON_EXISTING_COMPONENT_ID, FormulaGenerator
+
+logger = logging.getLogger(__name__)
+
+
+class EVChargerPowerFormula(FormulaGenerator):
+    """Create a formula engine from the component graph for calculating grid power."""
+
+    async def generate(self) -> FormulaEngine:
+        """Generate a formula for calculating total EV power from the component graph.
+
+        Returns:
+            A formula engine that calculates total EV charger power values.
+
+        Raises:
+            ComponentNotFound: if the component graph doesn't have a `GRID` component.
```

Comment:
This won't happen here,  because we are not looking for a grid component while generating this formula.

(I realize the other EV current formula has this in the docstring too.  I guess my vision is just sharper when I'm reading other peoples code. 😆  )

Reply:
Then I'll add a separate commit to fix that in EV current formula too

Reply:
done